### ISSUE TITLE
Unable to use this add-on on the latest NVDA version

### ISSUE THREAD

**Issue by pleveris** [fastfinge/eloquence_64#42]
Hello,

I got a new Lenovo PC running Windows 11. I've installed the latest version of NVDA (2025.3.2) and would like to use this add-on for having Eloquence synthesizer. However, I face the following issue right after installing this add-on and restarting NVDA (snippet from a log debug mode):
```bash
[...]
ERROR - synthDriverHandler.getSynthList (19:10:36.631) - MainThread (5512):
Error while importing SynthDriver eloquence
Traceback (most recent call last):
  File "synthDriverHandler.pyc", line 452, in getSynthList
  File "synthDriverHandler.pyc", line 439, in _getSynthDriver
  File "importlib\__init__.pyc", line 126, in import_module
  File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "C:\Users\PauliusLėveris\AppData\Roaming\nvda\addons\Eloquence\synthDrivers\eloquence.py", line 42, in <module>
    from . import _eloquence
  File "C:\Users\PauliusLėveris\AppData\Roaming\nvda\addons\Eloquence\synthDrivers\_eloquence.py", line 15, in <module>
    from multiprocessing.connection import Listener
  File "C:\Users\PauliusLėveris\AppData\Roaming\nvda\addons\Eloquence\synthDrivers\eloquence\multiprocessing\connection.py", line 30, in <module>
    import _multiprocessing
ImportError: DLL load failed while importing _multiprocessing: %1 is not a valid Win32 application.
DEBUGWARNING - synthDriverHandler.getSynthList (19:10:36.677) - MainThread (5512):
Synthesizer 'mssp' doesn't pass the check, excluding from list
[...]
```

Any ideas what could be a root cause, and what would be possible solutions/fixes?

Best regards,
Paulius

**Comment by pleveris** [comment]
OK, thank you, I have installed the latest Alpha build of NVDA. Now, after installing this add-on, it already appears in a list, but when I select it, the following error appears in NVDA log:

```bash
[...]
ERROR - synthDriverHandler.setSynth (09:30:31.326) - MainThread (13224):
setSynth failed for eloquence
Traceback (most recent call last):
  File "synthDriverHandler.pyc", line 514, in setSynth
  File "synthDriverHandler.pyc", line 480, in getSynthInstance
  File "C:\Users\PauliusLėveris\AppData\Roaming\nvda\addons\Eloquence\synthDrivers\eloquence.py", line 268, in __init__
    _eloquence.initialize(self._onIndexReached)
    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\PauliusLėveris\AppData\Roaming\nvda\addons\Eloquence\synthDrivers\_eloquence.py", line 373, in initialize
    response = _client.send_command("initialize", **payload)
  File "C:\Users\PauliusLėveris\AppData\Roaming\nvda\addons\Eloquence\synthDrivers\_eloquence.py", line 258, in send_command
    raise RuntimeError(response["error"])
RuntimeError: 'charmap' codec can't encode character '\u0117' in position 29: character maps to <undefined>
INFO - synthDriverHandler.setSynth (09:30:31.347) - MainThread (13224):
Falling back to previous synthDriver espeak
INFO - synthDrivers.espeak.SynthDriver.__init__ (09:30:31.358) - MainThread (13224):
Using eSpeak NG version 1.52.0
[...]
```

Any ideas?
